docs-samples/V4/JS/using-event-handlers/app.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Botbuilder v4 SDK - Using Event Handlers
+ * 
+ * This bot demonstrates how to use the sendActivity event handler
```

Comment:
Elaborate on this a bit more...what do you mean by "event handler"? Also, you say here "sendActivity" but in code you are using "sendActivities"....would be a good place to show the difference between the two and why you want to use one vs the other.